txpipe/source_selector.py

```diff
@@ -242,27 +228,31 @@ def apply_classifier(self, classifier, features, shear_data):
         """Apply the classifier to the measured magnitudes
         """
         bands = self.config['bands']
-        shear_prefix = self.config['shear_prefix']
 
         if self.config['shear_catalog_type'] == 'metacal':
-            variants = ['', '_1p', '_2p', '_1m', '_2m']
+            prefixes = ['mcal_', 'mcal_', 'mcal_', 'mcal_', 'mcal_']
```

Comment:
oh wait, you still have the metacal prefix... so in the input_cat.py file was the metacal section missing?

Reply:
The input_cat class only generates metadetect now, but we still support older metacal catalogs.